package.json

```diff
@@ -23,114 +23,113 @@
     "@material-ui/pickers": "4.0.0-alpha.12",
-    "@stripe/react-stripe-js": "^1.2.2",
-    "@stripe/stripe-js": "^1.11.0",
-    "@szhsin/react-menu": "^1.9.0",
+    "@stripe/react-stripe-js": "^1.4.1",
+    "@stripe/stripe-js": "^1.15.0",
+    "@szhsin/react-menu": "^1.10.1",
     "chart.js": "^2.9.4",
     "chartjs-plugin-datalabels": "^0.7.0",
-    "classnames": "^2.2.6",
-    "clipboard": "^2.0.6",
-    "formik": "2.2.6",
-    "framer-motion": "^3.2.1",
-    "lodash": "^4.17.20",
-    "luxon": "^1.25.0",
-    "notistack": "^1.0.3",
-    "postcss-focus-visible": "^5.0.0",
-    "react": "16.13.1",
+    "classnames": "^2.3.1",
+    "clipboard": "^2.0.8",
+    "formik": "^2.2.9",
+    "framer-motion": "^4.1.17",
+    "lodash": "^4.17.21",
+    "luxon": "^1.27.0",
+    "notistack": "^1.0.9",
+    "react": "^17.0.2",
     "react-chartjs-2": "^2.11.1",
-    "react-dom": "16.13.1",
+    "react-dom": "^17.0.2",
     "react-input-mask": "^2.0.4",
-    "react-modal": "^3.12.1",
+    "react-modal": "^3.14.3",
     "react-quill": "2.0.0-beta.2",
     "react-text-mask": "^5.4.3",
-    "yup": "^0.32.8"
+    "yup": "^0.32.9"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
+    "@babel/core": "^7.14.6",
     "@babel/helper-builder-react-jsx-experimental": "^7.12.11",
-    "@babel/plugin-proposal-class-properties": "^7.12.1",
-    "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
-    "@babel/plugin-proposal-private-methods": "^7.12.1",
-    "@babel/preset-env": "^7.12.11",
-    "@babel/preset-react": "^7.12.10",
-    "@babel/preset-typescript": "^7.12.13",
-    "@size-limit/preset-small-lib": "^4.9.2",
-    "@storybook/addon-a11y": "^6.2.9",
-    "@storybook/addon-actions": "^6.2.9",
-    "@storybook/addon-controls": "^6.2.9",
-    "@storybook/addon-docs": "^6.2.9",
-    "@storybook/addon-essentials": "^6.2.9",
-    "@storybook/addon-info": "^5.3.21",
-    "@storybook/addon-links": "^6.2.9",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.14.5",
+    "@babel/plugin-proposal-private-methods": "^7.14.5",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/preset-typescript": "^7.14.5",
+    "@size-limit/preset-small-lib": "^4.12.0",
+    "@storybook/addon-a11y": "^6.3.0-rc.7",
+    "@storybook/addon-actions": "^6.3.0-rc.7",
+    "@storybook/addon-controls": "^6.3.0-rc.7",
+    "@storybook/addon-docs": "^6.3.0-rc.7",
+    "@storybook/addon-essentials": "^6.3.0-rc.7",
+    "@storybook/addon-info": "^6.0.0-alpha.2",
+    "@storybook/addon-links": "^6.3.0-rc.7",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/addon-storyshots": "^6.2.9",
-    "@storybook/addon-storyshots-puppeteer": "^6.2.9",
-    "@storybook/addon-storysource": "^6.2.9",
-    "@storybook/addon-viewport": "^6.2.9",
-    "@storybook/addons": "^6.2.9",
-    "@storybook/react": "^6.2.9",
+    "@storybook/addon-storyshots": "^6.3.0-rc.7",
```

Comment:
why did we go from stable to RC versions? Is there a feature we're really wanting here?